modules/core/migrations/v5/migrations.go

```diff
@@ -0,0 +1,44 @@
+package v5
+
+import (
+	"strings"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	clientkeeper "github.com/cosmos/ibc-go/v3/modules/core/02-client/keeper"
+	clienttypes "github.com/cosmos/ibc-go/v3/modules/core/02-client/types"
+	host "github.com/cosmos/ibc-go/v3/modules/core/24-host"
+	"github.com/cosmos/ibc-go/v3/modules/core/exported"
+)
+
+// Localhost is the client type for a localhost client. It is also used as the clientID
+// for the localhost client.
+const Localhost string = "09-localhost"
+
+// MigrateToV5 prunes the 09-Localhost client and associated consensus states from the ibc store
+func MigrateToV5(ctx sdk.Context, clientKeeper clientkeeper.Keeper) {
```

Comment:
nit: we could potentially rename this function to make it more explicit as to what it does? (something like `Migrate_RemoveLocalHostClient`)